sourcegraph/cs-project
# MVP for CS interview

Part of: https://github.com/sourcegraph/sourcegraph/issues/21112 
How to test:
from the `/src/` folder run: `docker-compose up --build -d` and then navigate to localhost:8080. 
Notice that the items do not save after a page refresh. 

Next, modify the `docker-compose.yaml` file and change `DB_PORT: 5433` to `DB_PORT: 5432`. Items should now be persisted correctly.
